physionet-django/project/templatetags/project_templatetags.py

```diff
@@ -202,7 +202,7 @@ def call_method(obj, method_name, *args):
         args: arguments to be passed to the method
 
     Example:
-        {% call_method published_project 'has_access' request.user as user_has_access %}
+        {% call_method published_project 'can_view_files' request.user as user_has_access %}
```

Comment:
Not really an issue, but `user_has_access` is slightly off now that we have changed from `has_access` to `can_view_files`